### PR TITLE
Minor grammar and sentence changes

### PR DESCRIPTION
**Pull request by jeffreyhaines**

### CODE REVIEW

File: docs/source/openmw-cs/tour.rst [hunk 1]
```diff
@@ -3,8 +3,8 @@ A Tour through OpenMW CS: making a magic ring
 
 In this first chapter we will create a mod that adds a new ring with a simple
 enchantment to the game. The ring will give its wearer a permanent Night Vision
-effect while being worn. You don't need prior knowledge about modding
-Morrowind, but you should be familiar with the game itself. There will be no
+effect while being worn. You do not need prior Morrowind modding experience,
```
Comment (psi29a) [inline docs/source/openmw-cs/tour.rst]: You pull apart a contraction don't->do not, but you left prior?  Prior is a pompous term when previous would do just fine. :)

File: docs/source/openmw-cs/tour.rst [hunk 2]
```diff
@@ -13,7 +13,7 @@ installed and playable.
 Adding the ring to the game's records
 *************************************
 
-In this first section we will define what our new ring is, what it looks like
+In this first section we will define what our new ring is, what it looks like,
```
Comment (psi29a) [inline docs/source/openmw-cs/tour.rst]: That extra comma needs to go, it should not be there. The APA and MLA can go get stuffed, the rest of the English speaking world is against the serial comma.

File: docs/source/openmw-cs/tour.rst (hunk 1 above)
Comment (jdh6660) [inline docs/source/openmw-cs/tour.rst]: I agree.

File: docs/source/openmw-cs/tour.rst (hunk 2 above)
Comment (jdh6660) [inline docs/source/openmw-cs/tour.rst]: I think most people prefer the Oxford comma.

File: docs/source/openmw-cs/tour.rst [hunk 3]
```diff
@@ -58,7 +58,7 @@ detached panel can be re-attached to a window by dragging it by the title bar
 on top of the window.
 
 Now let's look at the panel itself: we have a filter text field, a very large
-table and a status bar. The filter will be very useful when we want to find an
+table, and a status bar. The filter will be very useful when we want to find an
```
Comment (psi29a) [inline docs/source/openmw-cs/tour.rst]: You added a comma when it wasn't needed and left 'an' before word beginning with a consonant. :/

File: docs/source/openmw-cs/tour.rst [hunk 4]
```diff
@@ -95,7 +95,7 @@ holding Shift to edit it (this is a configurable shortcut), but there is a
 better way: right-click the row of our new record and chose *Edit Record*, a
 new panel will open.
 
-We can right-click the row of our new record and chose *Edit Record*, a
+We can right-click the row of our new record and choose *Edit Record*, a
```
Comment (psi29a) [inline docs/source/openmw-cs/tour.rst]: Like you did above, perhaps select is a better word choice.

File: docs/source/openmw-cs/tour.rst (hunk 3 above)
Comment (jdh6660) [inline docs/source/openmw-cs/tour.rst]: Are you talking about "an entry"? I'm pretty sure e is a vowel.

File: docs/source/openmw-cs/tour.rst (hunk 4 above)
Comment (jdh6660) [inline docs/source/openmw-cs/tour.rst]: I agree.

File: docs/source/openmw-cs/tour.rst (hunk 1 above)
Comment (jeffreyhaines) [inline docs/source/openmw-cs/tour.rst]: I agree.

File: docs/source/openmw-cs/tour.rst (hunk 3 above)
Comment (jeffreyhaines) [inline docs/source/openmw-cs/tour.rst]: Are you referring to the "an entry"?

File: docs/source/openmw-cs/tour.rst (hunk 2 above)
Comment (psi29a) [inline docs/source/openmw-cs/tour.rst]: The Oxford comma is dead.

 http://www.adweek.com/galleycat/oxford-comma-dropped-by-university-of-oxford/34361?red=as

File: docs/source/openmw-cs/tour.rst (hunk 3 above)
Comment (psi29a) [inline docs/source/openmw-cs/tour.rst]: You're quite right, I must have been reading the following line incorrectly.

The unnecessary Oxford comma still stands though. :)

**Comment by psi29a** [comment]
A few fixups, then we can merge this in. Thank you again for reviewing the text. I'm glad there are more eyes on it! :)

**Comment by psi29a** [comment]
We should have a style-guideline... reading: No Oxord Commas. ;)

**Comment by jeffreyhaines** [comment]
Idk, I think you are just against it. I would think the best way to go about this is to pull the community, since that's who these guides are for anyway. Lets create a pull :)

**Comment by jeffreyhaines** [comment]
Removed the Oxford comma. At least we have the proper usage of choose and chose.

File: docs/source/openmw-cs/tour.rst (hunk 2 above)
Comment (jeffreyhaines) [inline docs/source/openmw-cs/tour.rst]: Oooooo 2 to 1 Oxford ;)

**Comment by psi29a** [comment]
Small fix during merge conflict, merging now.  Thank you! :)

Yes, I (and my journalist wife) are biased against the serial/oxford comma as it adds nothing to a list aside from cluttering it up.

**Comment by kcat** [comment]
In my mind, it helps to clearly delineate each thing in the list. After all, it wouldn't make sense to have a choice between three things like this: cereal, bacon and eggs and ham and cheese. It helps the statement to flow better with it, as each list entry is separated clearly and consistently, and identifies it as being a list rather than some kind of identification like this: the target, an Ascended Sleeper and Dagoth Velos (are the Ascended Sleeper and Dagoth Velos the target, or is the target something else that's with them?).

**Comment by psi29a** [comment]
@kcat You hit the exception, called a cluster which happens is most languages. The use of the comma is used to help differentiate between these and clusters. Even the BBC makes an exception here which would otherwise eschew any extraneous commas.

Example: http://content.tfl.gov.uk/tfl-editorial-style-guide.pdf
> Note: A comma is not required before and at the end of a list unless one of the listed items includes another ‘and’: We ordered chocolate cake, cheese and biscuits, and ice cream

**Comment by kcat** [comment]
@psi29a Right, but that's what I mean. Why only have it around for exceptions, when a situation that calls for it may not be immediately obvious? Or where one person may intuit the meaning without the extra comma while someone else may not? Why not just be consistent and avoid the problem entirely?

Additionally, having it adds structure. Not including the extra comma in my eye is similar to having a list like this:
* Entry one
* Entry two

Entry three. It visually removes the final item from the otherwise comma-separated list. Should we kill the Oxford bullet-point too? :P

**Comment by psi29a** [comment]
Considering that Oxford killed their own comma, we just need to wait around long enough for the rest to come full circle. :P

If you want to be internally consistent with how bullets are used, then drop the 'and' all together and use only commas exclusively to indicate individual items.

The Shatner and Walken commas are perfectly acceptable within context. ;)
http://liveoakcommunications.com/wp-content/uploads/2016/03/walken.png

**Comment by jeffreyhaines** [comment]
Let's follow what Oxford recommends, and use the comma when it removes ambiguity.

https://www.ox.ac.uk/sites/files/oxford/media_wysiwyg/University%20of%20Oxford%20Style%20Guide.pdf#page14

**Comment by psi29a** [comment]
I'm OK with this. They even go to say:

> Use semicolons in place of commas in a complicated list or sentence if it will improve clarity, particularly if list items already include commas.

This is inline what I posted above as well.

**Comment by jeffreyhaines** [comment]
Yeah I agree with what Oxford is recommending. But lets not use that as an excuse to write lists with semicolons. That should really only happen once if ever. We should be concerned with correct spelling, good writing and then technicalities.

**Comment by psi29a** [comment]
Agreed. :)

**Pull request merged**